docs/ruleset_functions/get_proposed_hvac_modeled_with_virtual_cooling.md

```diff
@@ -0,0 +1,40 @@
+# get_proposed_hvac_modeled_with_virtual_cooling
+
+**Description:** Get the list of HeatingVentilationAirAconditioningSystem in which Appendix G Table G3.1 #10 d is applicable.
+
+**Inputs:**
+- **U-RMR**: The U-RMR to determine if an HVAC system has been designed or modeled as existing and with cooling.
+- **P-RMR**: To determine if the same HVAC system has been modeled with cooling in the P-RMR.
+
+**Returns:**
+- **proposed_hvac_modeled_with_virtual_cooling_list_p**: A list that saves all the HVAC systems in which Appendix G Table G3.1 #10 d are applicable (i.e. modeled with virtual cooling in the proposed).
+ 
+**Function Call:** 
+
+1. match_data_element()
+
+**Logic:**
+- For each building_segment_p in the P_RMR..BuildingSegment: `for building_segment_p in P_RMR...BuildingSegment:`
+    - Get analogous segment in the U_RMR: `building_segment_u = match_data_element(U_RMR,BuildingSegment,building_segment_p.id)`
+    - Get list of hvac systems in the U_RMR: `hvac_list_u = building_segment_u.cooling_ventilation_air_conditioning_systems`
+    - For each hvac_p in the building_segment_p: `for hvac_p in building_segment_p.heating_ventilation_air_conditioning_systems:`   
+        # The line below will check if a virtual HVAC system was modeled in the proposed. If the hvac_p is not in the U_RMR then its a virtual system.
+        - Check if the hvac system in the proposed design model is NOT in the U_RMR: `if hvac_p.id not in hvac_list_u:`
+            - Reset virtual_cooling_modeled_check_p boolean variable: `virtual_cooling_modeled_check_p = FALSE`
+            # Below checks if a cooling system specifically was modeled (could just be heating for this hvac system) for the vitual system in the proposed. 
+            - For each cooling_system_p in hvac_p: `for cooling_system_p in hvac_p.cooling_system:`
+                - Check if the cooling_system_type != "None": `if cooling_system_p.cooling_system_type != "None":`
```

Comment:
not sure why the cooling system type will be = None for the proposed, it should only be none in the U RMR, correct?